libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -264,6 +264,11 @@ bool AP_RPM::get_rpm(uint8_t instance, float &rpm_value) const
     return true;
 }
 
+float AP_RPM::get_rpm_no_check(uint8_t instance) const
+{
+    return state[instance].rate_rpm;
+}
+
```

Comment:
Do we need the function for this? At least it should be private, this really should not be used anywhere except logging.